docs/developer-guide/plugins.md

```diff
@@ -54,7 +54,8 @@ const ruleFunction = (primary, secondaryOptions, context) => {
         ruleName,
         message: messages.rejected(selector),
         node: ruleNode,
-        word: selector
+        index: 0,
+        endIndex: selector.length
```

Comment:
We want to not recommend `word` but the previous code was actually better.
see https://github.com/postcss/postcss/blob/99da2f28aeded88c5e0fa3b4dd6fb44230710f57/lib/node.js#L261-L273

We would need to be certain that it's always `0` for us to make that change.
I think to leave it in for now, only `line` is being deprecated.
see https://github.com/stylelint/stylelint/issues/8126#issuecomment-2482487257